README.md

```diff
@@ -4,41 +4,58 @@
+on Earth, showcasing automated loading of data using [TypeDB-Loader.](https://github.com/typedb-osi/typedb-loader)
+- [GitHub](software/github/schemas/github-schema.tql) — load data from a live repository on GitHub or from a Vaticle GitHub snapshot, and get results via a custom GUI interface that
+uses the Java client to fetch the requested data.
+- [Phone Calls](telecom/phone_calls/schema.tql) — a database of customers of a fictional telecom company and calls they make.
+- [XCOM 2](gaming/xcom/schema.tql) — a database of interdependent research tasks in the game XCOM 2, featuring automatic inference of available
+research based on completed tasks and available items.
+
 ## Explore Client Examples
 
 TypeDB officially supports clients for Java, Node.js and Python. Learn more about [TypeDB Clients](http://docs.vaticle.com/docs/client-api/overview).
 
-### Client Java Examples
-- [Phone Calls: Migrate and perform expressive queries using Client Java](phone_calls/java/)
-- [XCOM Project: Migrate and perform logical queries on tree-structured game data using Client Java](xcom/)
+### Phone Calls
+
+[The Phone Calls example](telecom/phone_calls) showcases basic migration of phone call data in various formats (CSV, JSON, and XML)
+and performing expressive TypeQL queries using the [Java,](telecom/phone_calls/java) [Node.js,](telecom/phone_calls/nodejs) and
+[Python](telecom/phone_calls/python) clients.
+
+#### Data migration
+- Java: [CSV](telecom/phone_calls/java/CSVMigration.java) | [JSON](telecom/phone_calls/java/JSONMigration.java) | [XML](telecom/phone_calls/java/XMLMigration.java)
+- Node.js: [CSV](telecom/phone_calls/nodejs/migrateCsv.js) | [JSON](telecom/phone_calls/nodejs/migrateJson.js) | [XML](telecom/phone_calls/nodejs/migrateXml.js)
+- Python: [CSV](telecom/phone_calls/python/migrate_csv.py) | [JSON](telecom/phone_calls/python/migrate_json.py) | [XML](telecom/phone_calls/python/migrate_xml.py)
+
+#### Query examples
+- [Java](telecom/phone_calls/java/Queries.java)
+- [Node.js](telecom/phone_calls/nodejs/queries.js)
+- [Python](telecom/phone_calls/python/queries.py)
+
+### Github (GUI)
 
-### Client Node.js Examples
-- [Phone Calls: Migrate and perform expressive queries using Client Node.js](phone_calls/nodejs/)
+[The github example](software/github) showcases migration of heavily interconnected github repository data, and provides a visual
+interface to explore some sample queries.
 
-### Client Python Examples
-- [Phone Calls: Migrate and perform expressive queries using Client Python](phone_calls/python/)
+### XCOM Project
 
-## Explore Use Cases
+[The XCOM 2 example](gaming/xcom) showcases migration of tree-structured game data and logical inference capabilities of TypeDB.
+See [the schema](gaming/xcom/schema.tql) for the examples of reasoner rules inferring attributes.
 
-### Migrate data to TypeDB:
+## Explore TypeDB Loader Examples
```

Comment:
Hmmm, I'm interested in the idea of separating the examples by how they load the data, but I'm not sure that it's the strongest logical separation that exists for them.

I can think of two separate categorisations that users might be interested in:

- Execution environment (Python on MacOS, Node.js on Windows loading into Cluster, JVM on Raspberry Pi on a robot, etc.)
- Domain of dataset (biology, gaming etc)

The use of TypeDB Client or TypeDB-Loader is more of a low-level distinction - an implementation detail of the execution environment. Still interesting, but shouldn't be the primary way to differentiate project types. I would still definitely mention in the project description of _Catalogue of Life_ that we use Loader - which you already have done.

We already split by domain in the package structure, so this also seems like the right split for the README. But then, as we know, each domain only contains one project. So now what?

Taking a wider look at the README as a whole, we note that it's split into "Explore Schemas" and "Explore Client Examples" (before this PR). This was also in line with the old structure, where schemas were in a `schemas` package, with the applications in... other packages.

So I think it's time to re-align the entire README with the new structure:

- Dissolve the "Explore Schemas" section completely.
- Make each domain a top-level title - but because each domain only contains 1 project, make the project name _part_ of that title (e.g: `Gaming - XCOM Project`)
- Reduce titles to minimal clear version

The resulting structure will look like:
```md
# TypeDB Examples
## Biology: Catalogue of Life
## Gaming: XCOM Project
## Software: GitHub
## Telecom: Phone Calls
### Data migration
### Query examples
## Visualise data using TypeDB Studio
## TypeDB Bio
## TypeDB CTI
```
I'm not 100% happy with the fact that TypeDB Bio and TypeDB CTI are in the same "grouping" as the projects in Examples repo itself, but this seems 95% OK.

The content for TypeDB CTI can be found at https://github.com/typedb-osi/typedb-cti . I would make the description be:

TypeDB CTI is an open source threat intelligence platform for organisations to store and manage their cyber threat intelligence (CTI) knowledge. It enables threat intel professionals to bring together their disparate CTI information into one database and find new insights about cyber threats.